src/pykiso/exceptions.py

```diff
@@ -60,3 +60,15 @@ def __init__(self, aux_name: str) -> None:
         """
         self.message = f"A connector is required for {aux_name} and none is provided"
         super().__init__(self.message)
+
+
+class InvalidTestModuleName(PykisoError):
+    """Raised when a invalid python module name is covered by test file pattern
+    of either the cli or the yaml config
+    """
+
+    def __init__(self, name: str) -> None:
+        self.message = (
+            f"Test files need to be valid python module names but '{name}' is invalid"
```

Comment:
Also tell what are invalid symbols or characters. Or what is allowed
If i look into the pattern r'[_a-z]\w*\.py$'   
only lowercase words are allowed and the "_" char

Reply:
The pattern used by unittest also allows upper case (it is compiled with re.IGNORECASE) and numbers (\w). Added hint.